code/processes/dqe.q

```diff
@@ -56,6 +56,14 @@ nullchk:{[t;colslist;thres]
   update thresholdfail:nullspercentage>thres from res                                                           /- compare each column's nulls percentage with threshold thres
   }
 
+rangechk:{t;colslist;tlower;tupper;thres]                                                                       /- function to check percentage of values within tlower and tupper range
+  colslist:colslist except exec c from meta t where t in "csSC";                                                /- exclude columns that do not have pre-defined limits
```

Comment:
```suggestion
  colslist:(),colslist except exec c from meta t where t in "csSC";                                                /- exclude columns that do not have pre-defined limits
```

then can remove the need for the `(),` on all the subsequent lines & remove some brackets

Also there may be some other datatypes that should be excluded - should probably exclude `" "` for mixed columns, and I think these checks probably won't work on nested cols e.g. `"FJI"` etc.

Reply:
If `collist` can be an atom then I think you need this:
```
  colslist:((),colslist)except exec c from meta t where t in "csSC";
```
`except` expects first arg to be a list, if `collist` is always a list then this doesn't apply.